src/OSPSuite.Presentation/Presenters/ParameterIdentifications/MultipleParameterIdentificationFeedbackPresentersManager.cs

```diff
@@ -0,0 +1,52 @@
+﻿using OSPSuite.Core.Domain.ParameterIdentifications;
+using OSPSuite.Core.Events;
+using OSPSuite.Utility.Collections;
+using OSPSuite.Utility.Container;
+using OSPSuite.Utility.Events;
+
+namespace OSPSuite.Presentation.Presenters.ParameterIdentifications
+{
+   public interface IMultipleParameterIdentificationFeedbackPresentersManager : 
+      IListener<ParameterIdentificationStartedEvent>,
+      IListener<ParameterIdentificationTerminatedEvent>,
+      IListener<ProjectClosedEvent>
+   {
+      IParameterIdentificationFeedbackPresenter FeedbackPresenterFor(ParameterIdentification parameterIdentification);
+   }
+
+   public class MultipleParameterIdentificationFeedbackPresentersManager : IMultipleParameterIdentificationFeedbackPresentersManager
```

Comment:
Why is it called `MultipleParameterIdentificationFeedbackPresentersManager` and not `ParameterIdentificationFeedbackPresentersManager`